tutorials/text_processing/WFST_Tutorial.ipynb

```diff
@@ -0,0 +1,8368 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "Qq1Hz6CKWdwl",
+    "outputId": "3d8f5bd6-f10e-431d-9039-eb88164fbb95"
+   },
+   "outputs": [],
+   "source": [
+    "if 'google.colab' in str(get_ipython()):\n",
+    "  !pip install -q condacolab\n",
+    "  import condacolab\n",
+    "  condacolab.install()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "T0JxcvuPHvn9"
+   },
+   "source": [
+    "NeMo's Inverse Text Normalization (ITN) module uses Weighted Finite State Transducers (WFST) to deploy grammars for efficient text downstream ASR processing. In this tutorial, you will learn to build a normalization grammar from the ground up to use in your own text processing tasks."
```

Comment:
let's start general? not just restricted to inverse text normlizatoin? when you explain you can say you will showcase using example of itn, but the principles can be used for both tn and itn